frontend/src/App.tsx

```diff
@@ -1,24 +1,23 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import axios from "axios";
+import { useState } from "react";
+const serverURL = 'http://localhost:8000'
 
 function App() {
+  const [message, setMessage] = useState("");
+  
+  const handleClick = async () => {
+    if (message) {
+      setMessage("")
+      return
+    }
+    const res = await axios.get(`${serverURL}/api`)
+    const serverMessage = res.data.data
+    setMessage(serverMessage)
+  }
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Hello Wrap
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h2>{message ? message : ""}</h2>
```

Comment:
no need to change but this can also be written as `message ?? ""`

this is a relatively recent addition to js and it's super helpful for these kinds of patterns
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
Fixed line 19 to `message ?? ""`. Pretty useful ngl.